sw/device/lib/drivers/rv_plic/rv_plic.c

```diff
@@ -49,6 +49,7 @@
 #include "i2s.h"
 #include "dma.h"
 #include "spi_host.h"
+#include "iffifo.h"
```

Comment:
this should never be done, HEEP is used in other project without the testbench, so the plic functions should stay sane - move these parts in the application file

Reply:
Switched to `plic_assign_external_irq_handler` implementation instead.